From 0fd88d398896353074fee153259dbf3530ca423f Mon Sep 17 00:00:00 2001
From: Anton Khirnov <anton@khirnov.net>
Date: Thu, 12 Jan 2012 09:12:59 +0100
Subject: [PATCH 16/67] riff: remove references to sonic codec ids

They are deprecated and will be purged on next major bump.
---
 libavformat/riff.c |    2 --
 1 files changed, 0 insertions(+), 2 deletions(-)

diff --git a/libavformat/riff.c b/libavformat/riff.c
index 8489bc7..4caac80 100644
--- a/libavformat/riff.c
+++ b/libavformat/riff.c
@@ -327,8 +327,6 @@ const AVCodecTag ff_codec_wav_tags[] = {
     { CODEC_ID_AAC_LATM,        0x1602 },
     { CODEC_ID_AC3,             0x2000 },
     { CODEC_ID_DTS,             0x2001 },
-    { CODEC_ID_SONIC,           0x2048 },
-    { CODEC_ID_SONIC_LS,        0x2048 },
     { CODEC_ID_PCM_MULAW,       0x6c75 },
     { CODEC_ID_AAC,             0x706d },
     { CODEC_ID_AAC,             0x4143 },
-- 
1.7.5.4

